package.json

```diff
@@ -12,45 +12,47 @@
     "index.d.ts"
   ],
   "scripts": {
-    "lint": "eslint es/**/*.mjs spec/**/*.spec.mjs",
+    "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "NODE_OPTIONS=--es-module-specifier-resolution=node jasmine",
+    "test": "ts-node --project tsconfig-cjs.json node_modules/jasmine/bin/jasmine",
     "prebuild": "npm run test",
-    "build": "rm -rf lib && babel es -d lib/esm --source-maps && babel es -d lib/cjs --config-file ./babel.config.cjs.json --source-maps && ./fixup",
+    "build": "rm -fr dist/* && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
     "prerelease": "npm run build",
     "release": "npm version -m \"New version: %s\"",
     "postrelease": "npm run push && npm publish",
     "push": "git push origin master && git push origin --tags",
     "prepare": "husky install"
   },
   "lint-staged": {
-    "*.+(js|mjs)": [
+    "*.ts": [
       "eslint --cache --fix",
       "prettier --write"
-    ],
-    "*.js": "eslint --cache --fix"
+    ]
   },
   "dependencies": {
-    "@babel/runtime": "^7.15.4",
     "is-retry-allowed": "^2.2.0"
   },
+  "peerDependencies": {
+    "axios": "^0 || ^1"
```

Comment:
I could test it later but I have feeling that `^0` might not work as expected. In the 0.x.x range the caret has different meaning - it allows to only bump within "patch" range. So I'm not sure what would specifying it that way actually mean.

You could for example change it to `0.x || 1.x` which I think would be also good.

Reply:
No need to check, you are 100% correct!